notificationservice.cpp

```diff
@@ -46,6 +46,7 @@ public slots:
                     else if(mReader.name() == "ai") appIcon = mReader.readElementText();
                     else if(mReader.name() == "su") summary = mReader.readElementText();
                     else if(mReader.name() == "bo") body = mReader.readElementText();
+                    else if(mReader.name() == "vb") vibrate = mReader.readElementText();
```

Comment:
Also if we at some point have more user options à la cyanogenmod where the user can specify separately colors of LEDs, sounds, screen flashing, whatever, it might make sense to have the decoding of `vibrate+led+whatever` to NGFD profiles here anyways.